aws/canary/heartbeat/python/canary.py

```
@@ -24,4 +23,4 @@ def main():
 
 def handler(event, context):
```

Ruff

Missing return type annotation for public function `handler`

(ANN201)